### ISSUE TITLE
Create <SeriesHero /> Component

### ISSUE THREAD

**Issue by michael-martin-al** [lifechurch/youversion-web-open-ideas#16]
## Summary
Build a component for displaying a video series hero. In the attached screenshot, you can see what I'm m talking about. Your job is to create a very similar user interface using only components from our `melos` component library: https://github.com/lifechurch/melos

You won't be able to recreate the exact look and feel that you see on [the Bible.com videos page](https://www.bible.com/videos/266/series), but that's OK! 😄 What you see on Bible.com represents our old design style, but `melos` represents our newest design specs. I am sure you'll have questions, so feel free to ping me. We'll get one of our YouVersion designers to give you the final 👍  when you're done. So, you can expect there to be some back and forth work with them getting it just right.

## Mock API Data
After #11 has been completed, there should be some mock data available to use while building your component. Instead of hard-coding the strings and images, you want to grab those from the mock api data. If you have questions about this, just make a comment here and we'll do our best to answer quickly.

## This is what you're building
![image](https://user-images.githubusercontent.com/3265272/38436789-32f62020-399b-11e8-9683-7ddfd187bd35.png)

**Comment by nathanbedford** [comment]
@michael-martin-al I'd be happy to take a crack at this one

**Comment by nathanbedford** [comment]
Is the social section in the lower right a separate component (existing or new)?

**Comment by michael-martin-al** [comment]
@nathanbedford  We're using AddThis for that stuff. It's not part of this issue. We'll tackle it in a separate issue. Thanks for taking on this issue.

**Comment by nathanbedford** [comment]
Hi @michael-martin-al, I apologize for the delay on this card...it's been a busy week work-wise for me.  I'm starting to focus on this now, and I have a few questions, in particular related to which elements of melos to use.  I'll reference the sections of the component via the number code in the screenshot below:

![2018-04-14 14_57_33-easter - lightworkers jesus film and lumo _ videos _ the bible app _ bible com](https://user-images.githubusercontent.com/195696/38771706-b003c008-3ff5-11e8-91da-4a973e921e6e.png)

Obviously I'll need some sort of container (**1**) but I didn't think the Card component was a good fit for this, since it would adding padding around the div.  Thoughts?

I looked into using LazyImage for (**2**) but I'm thinking it may be difficult to get the current effect (I referenced [https://www.bible.com/videos/266/series](the Easter series)) with LazyImage since it's using an img tag vs a background image (specifically using the background-size:cover).  If you guys want to stick with LazyImage I could explore that ('object-fit' CSS property, etc). 

(**3**) is current an h1 in the existing site, so I could use the Heading1 component from melos.

(**4**) looks to be just a background image (svg), current referenced relatively.  Shall I copy that asset into this project and reference relatively?

(**5**) is currently an h2, but if I use Heading2 from melos that text gets centered.  Want me to still use Heading2? I'm trying to understand the full vision of melos :)

Also, (**5**) is a link so I'm planning on using LinkText (unless that's more designed for links in content).

Lastly, I noticed this Series Hero currently is hidden at smaller breakpoints - I'm assuming that is something you guys want handled outside the component.  Correct me if I'm wrong.

**Comment by michael-martin-al** [comment]
1. Agreed. `melos` `<Card />` doesn't really work here. Are you familiar with `glamorous` and how it works? You can do something like this:
```
import { Div } from 'glamorous'

// and then
// these are just arbitrary values to give you an example
<Div margin={10} width={'100%'} />
```

Using `glamorous` allows us to only deliver the exact CSS we need on a page and deliver in various ways (inline, external sheet, etc.)

2. Agreed here as well. `<LazyImage />` doesn't really make sense. Plus, it's going to be at the top of the page and immediately in the viewport so lazy loading doesn't make a lot of sense. 

3. Yes, `<Heading1 />` looks right to me. Especially for SEO purposes, it will be good to have the most important title on the page in an `<h1 />`. You'll likely have to add some custom css to make it white, or figure out how the `<Theme />` works in `melos`. 

4. So, what we've done in the past is import svg icons as React components. That makes them more flexible and able to respond to props in the future. Here's an example: https://github.com/lifechurch/melos/blob/master/src/components/icons/XMark/index.js

5. 😄 You can pass a `textAlign` prop and set it to `left` on the `<Heading2 />` component. I will explain the full vision of `melos` below. And, yes to using `<LinkText />`.

## The Vision of `melos`
We've been building user interfaces in React for a couple of years, but in a very ad-hoc manner. Naturally, certain UI elements started getting used over and over in different features, so we migrated these components up into a `components` folder in our main repo. But, they were difficult to discover and poorly documented so we ended up building duplicate components with slightly different properties and UI but intended for similar purposes. This was a beast to manage. So, we started trying to standardize our user interfaces. At the same time, one of our designers came up with a basic web style guide. So, we married our components folder with the style guide to create a standard for user interfaces in all web projects and created the `melos` repo. The intent is to keep our UI D.R.Y (don't repeat yourself 😏). I want all headings to be rendered consistently across all pages. The same for cards and links and everything else. But, alas `melos` is still in its infancy and you are being asked to use it as if it were some mature UI library. Thus, the questions and lack of clarity on our part.

**Comment by nathanbedford** [comment]
@michael-martin-al, would you like the image (2) to get passed to the component as a prop (string url) or is there something else you had in mind?

Also, regarding the CSS I'm doing for this, should that all be via glamorous, or is there ever a case for inline styles? We are avoiding separate CSS files, right?

**Comment by michael-martin-al** [comment]
@nathanbedford Sorry for the delayed response.
Yes, let's pass in the image url as a string prop. As for CSS, I would prefer glamorous. You can define it in almost the same way as defining inline-styles in React. But, we get the added benefit of rendering that CSS in different ways. Plus, it auto-prefixes and minifies.

**Comment by nathanbedford** [comment]
@michael-martin-al  no problem, I very much understand.  Glamorous seems great! I'll put my other questions in the PR
